app/src/common/chromium/com/igalia/wolvic/browser/api/impl/CrashReporterServiceImpl.java

```diff
@@ -0,0 +1,17 @@
+package com.igalia.wolvic.browser.api.impl;
+
+import androidx.annotation.NonNull;
+
+import com.igalia.wolvic.browser.api.WRuntime;
+import com.igalia.wolvic.crashreporting.CrashReporterService;
+
+import org.mozilla.geckoview.GeckoRuntime;
```

Comment:
Not here :)

Reply:
Ah, forgot to delete that for Chromium when I copied this from Gecko implementation.